client/reader/search-stream/index.jsx

```diff
@@ -135,9 +187,22 @@ const FeedStream = React.createClass( {
 	},
 
 	render() {
+		var suggestions = null;
+		var lang = i18nUtils.getLocaleSlug();
+		if ( this.searchSuggestions[lang] ) {
+			suggestions = shuffle( this.searchSuggestions[lang] ).slice( 0, 3 );
```

Comment:
lodash's `sampleSize` would simplify this. Just grabs `n` elements from the array.